### ISSUE TITLE
Command line option -a not supported

### ISSUE THREAD

**Issue by tuky** [commitizen/cz-cli#376]
The readme says, that all commit options are supported, especially `-a`. But especially `-a` is **not** supported. Running `git cz -a` first adds all files and then executes the commit with `-a`. This will mess up repos, if you don't know that (and how could you know?).

**Comment by derikb** [comment]
Yeah, this one really threw me off, when I just started using commitizen. Very confusing.

**Comment by derikb** [comment]
For anyone who finds this... the alternative to trying `git cz -a` is:
```
$ git add -u
$ git cz
```
That will just add updated/deleted files rather than all untracked files.

**Comment by Pawamoy** [comment]
Agree on this, it has not the same effect as `git commit -a`. If I quit the committing process of `git commit -a`, it leaves without changing anything. If I do while `git cz -a`, files will have been added with `git add`. And not just modified files, every new file as well :confused:.

This is not a desirable behavior. Nothing should be staged if we `Ctrl-C` while `git cz -a`.

**Comment by LinusU** [comment]
Yeah, this seems like a bug to me or at least surprising behavior. PR welcome!
